elasticsearch/src/test/java/org/apache/calcite/adapter/elasticsearch/BooleanLogicTest.java

```diff
@@ -120,7 +118,6 @@ public static void setupInstance() throws Exception {
     assertEmpty("select * from view where num > 42 and num < 42 and num = 42");
     assertEmpty("select * from view where num > 42 or num < 42 and num = 42");
     assertSingle("select * from view where num > 42 and num < 42 or num = 42");
-    assertSingle("select * from view where num > 42 or num < 42 or num = 42");
```

Comment:
Instead of removing completely the test can you please skip the line using a (new) reference from https://github.com/apache/calcite/blob/master/core/src/main/java/org/apache/calcite/util/Bug.java.

Reply:
Done.